### PR TITLE
Get better lesson 7

### PR DESCRIPTION
**Pull request by PaulNSkiba**

### CODE REVIEW

File: LECT_10 (Google Storage)/dags/gcloud.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+from google.oauth2 import service_account
+
+import json
+import os
+import tempfile
+
+def upload_blob(bucket_name, source_file_path, destination_blob_name):
```
Comment (v-hunt) [inline LECT_10 (Google Storage)/dags/gcloud.py:21]: I expect students to find and use LocalFilesystemToGCSOperator operator. 
This function looks like reinventing a wheel.

File: LECT_10 (Google Storage)/dags/gcloud.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+		source_file_path='/opt/airflow/gcloud/src1_sales_2022-08-01__01.csv',
+		destination_blob_name='src1/sales/v1/2022/08/01/src1_sales_2022-08-01__01.csv',
+	)
+
+
+def run_job2():
+	upload_blob(
+		bucket_name='de2022-paul_skyba_bucket_1',
+		source_file_path='/opt/airflow/gcloud/src1_sales_2022-08-02__01.csv',
+		destination_blob_name='src1/sales/v1/2022/08/02/src1_sales_2022-08-02__01.csv',
```
Comment (v-hunt) [inline LECT_10 (Google Storage)/dags/gcloud.py:67]: dates in the path should be generated dynamically by Airflow on the run time.
IOW you should use a templated variable`{{ dag_run.logical_date }}`. More details here: https://airflow.apache.org/docs/apache-airflow/stable/templates-ref.html

File: LECT_10 (Google Storage)/dags/gcloud.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+task2 = PythonOperator(
+    dag=dag,
+    task_id='copy2',
+    python_callable=run_job2,
+    start_date=datetime(2022, 8, 2),
+    end_date=datetime(2022, 8, 3)
+)
+
+start >> [task1 , task2]
```
Comment (v-hunt) [inline LECT_10 (Google Storage)/dags/gcloud.py:105]: Actually, we should use the 1 task scheduled for 2 days of run in the DAG.

**Review by v-hunt — COMMENTED** [review]
Please, see my comments

File: LECT_10 (Google Storage)/dags/gcloud.py (hunk 1 above)
Comment (PaulNSkiba) [reply LECT_10 (Google Storage)/dags/gcloud.py:21]: Is there any mention of the "not-wheel" function in the list of homework steps? The question for the future is for resolving any "expections" concerning decreasing of the my HW's mark.

File: LECT_10 (Google Storage)/dags/gcloud.py (hunk 2 above)
Comment (PaulNSkiba) [reply LECT_10 (Google Storage)/dags/gcloud.py:67]: Got it.

File: LECT_10 (Google Storage)/dags/gcloud.py (hunk 3 above)
Comment (PaulNSkiba) [reply LECT_10 (Google Storage)/dags/gcloud.py:105]: "Should" or "May/Could"? There wasn't any strict demands in the Homework, actually. So, 50/50 "error".

**Pull request merged**